src/lib/environment/mod.rs

```diff
@@ -31,13 +31,24 @@ use rust_info::types::{RustChannel, RustInfo};
 use std::env;
 use std::path::{Path, PathBuf};
 
-fn evaluate_env_value(env_value: &EnvValueScript) -> String {
+fn evaluate_env_value(key: &str, env_value: &EnvValueScript) -> String {
     match command::run_script_get_output(&env_value.script, None, &vec![], true, Some(false)) {
         Ok(output) => {
             let exit_code = output.0;
             let stdout = output.1;
-
-            command::validate_exit_code(exit_code);
+            let stderr = output.2;
+
+            if exit_code != 0 {
+                error!(
+                    concat!(
+                        "Error while evaluating script for env: {}, exit code: {}\n",
+                        "script: {:#?}\n",
```

Comment:
can we change this one to
script:\n{:#?}\n

Reply:
Sure, np.